src/Files.Shared/Enums/SortOption.cs

```diff
@@ -7,9 +7,9 @@ public enum SortOption : byte
 		DateCreated,
 		Size,
 		FileType,
-		SyncStatus,
 		FileTag,
-		OriginalFolder,
-		DateDeleted
+		SyncStatus, // Cloud drive
```

Comment:
Why were these reordered?

Reply:
This was to separate the options available for all folders from those specific to recycle bin or cloud drives (which need to be hidden from global settings), but I can revert this changes as well as those about default sort direction

Reply:
That would be preferred 👍